helm/templates/dashboard.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.grafanaDashboard -}}
```

Comment:
Let's make this `.Values.grafanaDashboard.enabled` - that way the `.Values.grafanaDashboard` object can be easily extended with other options in backwards-compatible way.